src/main/java/main/CHScrobbler.java

```diff
@@ -64,6 +64,29 @@ public static void main(String[] args)
             else
                 dataFolder = prop.getProperty(Statics.DATA_FOLDER_PROP_NAME);
 
+            //validate scrobble threshold seconds
+            int scrobbleThresholdSeconds = 25;
+            String scrobbleThresholdSecondsString = prop.getProperty("scrobble_threshold_seconds");
+
+            if (scrobbleThresholdSecondsString != null && !scrobbleThresholdSecondsString.isEmpty())
+            {
+                try
+                {
+                    scrobbleThresholdSeconds = Integer.parseInt(scrobbleThresholdSecondsString);
+                }
+                catch(NumberFormatException e)
+                {
+                    scrobbleThresholdSeconds = -1;
+                }
+
+                if (scrobbleThresholdSeconds < 1 || scrobbleThresholdSeconds > 240)
+                {
+                    scrobbleThresholdSeconds = 25;
```

Comment:
Maybe consider making a `DEFAULT_SCROBBLE_THRESHOLD` in `Statics.java` and using it here?